src/build.sh

```diff
@@ -110,6 +112,11 @@ function build_kernel_main()
     parallel_cores="$(grep -c ^processor /proc/cpuinfo)"
   fi
 
+  if [[ -n "$cflags" ]]; then
+    build_cflags "$flag" "$output_kbuild_flag" "$parallel_cores"
```

Comment:
Correct me if I'm wrong, but, from my understanding, the way this is implemented makes building the kernel with CFLAGS sort of limited.
For instance, suppose I want to build a kernel with compilation warnings enabled and CFLAGS set. By calling this function, instead of appending the flags to the `command` variable, it would ignore any other option that I set -- in this case `warning_level`.

Reply:
@aquilamacedo Was this addressed?

Reply:
Yes, could you check if everything is in order? @magalilemes

Reply:
It does work now, so I'd just echo Siqueira's comment on just concatenating `KCFLAGS` to that line he mentioned. After you do that, I'll retest this PR.